### ISSUE TITLE
Relative git ssh module having ssh:// appended

### ISSUE THREAD

**Issue by typhoon2099** [pypa/pipenv#2441]
I'm trying to install my Pipenv environment on a new machine and have upgraded pipenv from 2018.05.18 to 2018.06.25. A module which I'm pulling from git is now no longer installing with the following error:

```
Command "git clone -q ssh://git@bitbucket.org:redacted/redacted.git /home/redacted/.local/share/virtualenvs/redacted-eQF46Ow3/src/redacted" failed with error code 128 in None
```

Pipenv appears to be appending `ssh://` to the URL where it previously hasn't.

------------

##### Expected result

The module is installed

##### Actual result
The module is not installed

##### Steps to replicate

Try to install the module

EDIT: I've investigated trying to install the module via the command line and get the following output:

Relative Git URL
Command:
```
pipenv install -e git+git@bitbucket.org:company/library.git@1.0.0-rc2#egg=libb
```

Error:
```
Command "git clone -q ssh://git@bitbucket.org:company/library.git /tmp/tmpENxA1wsource/library" failed with error code 128 in None
```

Absolute Git URL
Command:
```
pipenv install -e git+git@bitbucket.org/company/library.git@1.0.0-rc2#egg=libb
```

Error:
```
Command "git clone -q git@bitbucket.org/company/library.git /home/employee/.local/share/virtualenvs/library-joI2jbUq/src/libb" failed with error code 128 in None
```

It looks like the `ssh://` is being appended to the relative URL instead of the absolute URL.

**Comment by uranusjr** [comment]
Fixing one bug, causing another 💥 

We need to be smarter deciding how to fix the SSH URL.

**Comment by techalchemy** [comment]
Closed by #2453
